products/dependencytrack.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Dependency-Track
+category: server-app
+permalink: /dependencytrack
+alternate_urls:
+-   /dependency-track
+eolColumn: Security Support
+eolWarnThreshold: 121
+releaseColumn: Latest
+auto:
+  - git: https://github.com/apache/maven.git
```

Comment:
Please fix this to the correct repository.

Reply:
Fixed

Reply:
Still pointing to apache maven.

Reply:
@captn3m0  Sry, somehow I got confuse here.
Should be done now